client/src/components/views/AdminPage/ManageUsersPage/AdminUsersPage.tsx

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable react/prop-types */
```

Comment:
What's that?

Reply:
There is a problem with prop types of cell. Eslint shows message like this:
![image](https://user-images.githubusercontent.com/87782933/169570636-5db6dd90-2f7a-4ec4-96c1-c8cfced683c2.png)
 I have read, that it's the way to bypass this problem and just haven't found better solution.